client/src/pages/trainer/courses/list/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { Fragment, useState } from 'react';
+import type { ReactNode } from 'react';
+import Dropdown from '@/src/shared/components/Dropdown';
+import SearchBar from '@/src/shared/components/SearchBar/SearchBar';
+import GridIcon from '@/src/shared/icons/GridIcon';
+import ListIcon from '@/src/shared/icons/ListIcon';
+import ViewAs from '@/src/sections/courses/list/view-as';
+import RightSideContainer from '@/src/sections/courses/list/right-side-container';
+
+const View = (): ReactNode => {
+  const [selectedView, setSelectedView] = useState('grid');
+  const [gridIconColor, setGridIconColor] = useState('stroke-blue-500');
+  const [listIconColor, setListIconColor] = useState('');
+  const sample = [
+    'a',
+    'b',
+    'c',
+    'e',
+    'f',
+    'g',
+    'h',
+    'a',
+    'b',
+    'c',
+    'e',
+    'f',
+    'g',
+    'h'
+  ];
```

Comment:
- this array data is useless. Just create an empty array.
- use proper variable naming
```suggestion
  const data = Array(5).fill();
```